radio/src/datastructs.h

```diff
@@ -980,7 +980,11 @@ PACK(struct RadioData {
   NOBACKUP(int8_t backlightMode);
   NOBACKUP(TrainerData trainer);
   NOBACKUP(uint8_t view);            // index of view in main screen
-  NOBACKUP(int8_t buzzerMode:2);    // -2=quiet, -1=only alarms, 0=no keys, 1=all
+#if defined(BUZZER)
+  NOBACKUP(int8_t buzzerMode:2);    // -2=quiet, -1=only alarms, 0=no keys, 1=all (only used on AVR radios without audio hardware)
```

Comment:
Hmm these are actually completely redundant and could be ditched for all radios....
There's even some dead code in the only place it's used...

```
#if defined(BUZZER) // AUDIO + BUZZER
      case ITEM_SETUP_BUZZER_MODE:
        g_eeGeneral.buzzerMode = editChoice(RADIO_SETUP_2ND_COLUMN, y, STR_BUZZER, STR_VBEEPMODE, 

[...]

#elif defined(BUZZER) // BUZZER only
      case ITEM_SETUP_BUZZER_MODE:
        g_eeGeneral.beepMode = editChoice(RADIO_SETUP_2ND_COLUMN, y, STR_SPEAKER, STR_VBEEPMODE, g_eeGeneral.beepMode, -2, 1, attr, event);
```